### PR TITLE
Feature/template one

### PR DESCRIPTION
**Pull request by christocarr**
Capture

### CODE REVIEW

**Comment by pkijowska** [comment]
hey Chris, the section between your name and professional summary looks messy. Is there a way to fix it?

File: src/features/resume/FinalView.js [hunk 1]
```diff
@@ -1,72 +1,88 @@
...
-						</p>
-						<p className="header__text">
-							{linkedin}
-						</p>
+		<div className='wrapper'>
+			{pickedTemplate === `first-template` ? (
+				<TemplateOne pickedTemplate={pickedTemplate} resumeData={resumeData} />
+			) : (
+				<div className={pickedTemplate}>
+					<h5 className={`${pickedTemplate}__name`}>
```
Comment (pkijowska) [inline src/features/resume/FinalView.js:33]: in this case pickedTemplate doesn't make sense if it the other template is abstracted from the main view

**Review by pkijowska — CHANGES_REQUESTED** [review]

**Comment by christocarr** [comment]
> hey Chris, the section between your name and professional summary looks messy. Is there a way to fix it?

It's just a template. We are steering away from the essentials. It can be fixed later.

File: src/features/resume/FinalView.js (hunk 1 above)
Comment (christocarr) [reply src/features/resume/FinalView.js:33]: Could you elaborate please? Is this the template as a component? Your template should also be made into a component shouldn't it?

Comment (pkijowska) [reply src/features/resume/FinalView.js:33]: I don't mind doing it, no problem and I understand. However, usually the best practice is to abstract the whole file if one starts cleaning up not just part of it, as there is no such thing as "yours" and "mine" when working at a company. We are all helping each other here and trying to make our life easier when started an idea. I hope you understand what I mean.

Comment (christocarr) [reply src/features/resume/FinalView.js:33]: Okay, I'll merge then.

**Pull request merged**